pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
     <div class="container">
       <h1 class="title has-text-centered is-size-3 is-size-2-tablet is-size-1-desktop">
-        The Web — <a href="{{site.baseUrl}}/take3" title="Why Take 3?">Take 3</a>
+        Solid
       </h1>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-        Solid is a technology, like the Web, but a new level of standard which adds to the existing protocols to make it more powerful, particularly to empower individuals at home and at work.
+      Using Web standards to
+      let people control their data, and choose the
+      applications and services to use with it.
       </p>
+      <div class="buttons is-centered">
+        <!--
+        <a href="{{site.baseurl}}/about/">
+          <button class="button is-primary">About Solid</button>
+        </a>
+        -->
+        <a href="{{site.baseurl}}/users/">
+          <button class="button is-primary">Get a Pod</button>        
+        </a>
+        <a href="{{site.baseurl}}/developers/tutorials/getting-started">
+          <button class="button is-primary">Developer Tutorial</button>
+        </a>
+      </div>
     </div>
   </div>
 </section>
 
 <div id="landing-content">
-  {%
-    include choose-your-own-adventure.html
-      heading="Solid is great for…"
-      link1="/use-solid"
-      adventure1="Everyone"
-      link2="/for-developers"
-      adventure2="Developers"
-      link3="/for-enterprises"
-      adventure3="Enterprises"
-  %}
 
-  <div id="tour">
-    <section class="section">
-      <div class="container">
+  <section id="overview" class="section">
+
+    <div class="container">
+
+        <p class="subtitle has-text-centered is-3">
+          Solid lets people store their data in decentralized data stores 
+          called <b>pods</b>.
+        </p> 
+
         <div class="columns">
-          <div class="column is-hidden-touch is-offset-1 is-4">
-            <figure class="image">
-              <img src="{{site.baseurl}}/assets/img/single-sign-on.svg" alt="[]" />
-            </figure>
-          </div>
-          <div class="column is-offset-1 is-5">
-            <h2 class="title">The last login you'll ever need</h2>
-            <p class="content is-large">
-              Solid provides for the first time a <b>single global logon system</b>, so that when you log into any web site, instead of having  to log in with the usual 'f' and 'g', etc, blue buttons, and then be tracked by Facebook, Google, or some other large social network, instead you can log in with any Solid provider you trust, and that won't track you.
+          <div class="column is-half-desktop is-offset-one-quarter-desktop">
+          
+            <p class="content has-text-centered is-large">
+              Pods are like secure personal web servers for data.
+              All data in a pod is accessible
+              accesible via the 
+              <a href="https://solid.github.io/specification/">Solid Protocol</a>.
```

Comment:
Do we need _another_ word for this? I don't think I've ever seen "Protocol" used yet...

Reply:
This is something that was agreed upon by Editors and named by Tim. The main client/server specification is going to be named in the specification as the "Solid Protocol".

Reply:
That's good to know. So basically, almost everywhere we've been using "Solid standard(s)" and "Solid specification(s)" before, we should be using "Solid Protocol" from now on?

Reply:
Solid Specification is still good as an umbrella term for all pertinent specifications underneath. For example, Solid Protocol is the core client/server specification, but wouldn't include the spec for application interoperability. Solid Specification would encompass them both.

Reply:
Right, thanks. I think that doesn't apply for most of our uses of it in e.g. our documentation, so I'll feed that back to the relevant people.